esp/src/package.json

```diff
@@ -16,10 +16,10 @@
     "copy-res-stub_htm": "cpx \"./stub.htm\" ./build/",
     "copy-res": "run-p copy-res-es6-promise copy-res-eclwatch-img copy-res-eclwatch-ecl copy-res-dojo copy-res-dojox copy-res-TopoJSON copy-res-font-awesome copy-res-stub_htm",
     "compile": "tsc",
-    "compile-watch": "run-s compile -- -w",
+    "compile-watch": "npm run compile -- -w",
```

Comment:
These are probably by mistake but not needed in session fix.

Reply:
There ok to include.